### ISSUE TITLE
Is there a ios code for ProjectRTC?

### ISSUE THREAD

**Issue by jockyxiao** [pchab/ProjectRTC#34]
Is there a ios code for ProjectRTC?

**Comment by syedmaazali** [comment]
i have tested this with android rtc and its working perfectly fine but i also want to know is there any IOS project which is compatible ProjectRTC

**Comment by panchanisoft** [comment]
Yes, we have developed & tested iOS client with this projectRTC server. Please connect us at hiropanchani@gmail.com for more details.

**Comment by hoang89yenthe** [comment]
Please check my mail @panchanisoft  .:) .

**Comment by manishjss** [comment]
We are developing app for video call. We make it for android but ios we are facing problem. Can you help us or provide ios client.

**Comment by DennisRitchieUA** [comment]
i have iOS client for ProjectRTC server. contacts me xa4ik@i.ua

**Comment by digixtechnology** [comment]
https://github.com/digixtechnology/iOSRTC

**Comment by Aviru** [comment]
I am running ProjectRTC as server and using iOSRTC for ios. But encountered with the following errors:- 
ERROR SocketEnginePolling: Could not connect to the server.
ERROR SocketEngine: Could not connect to the server.
ERROR SocketIOClient: Could not connect to the server.
LOG SocketIOClient: Handling event: error with data: ["Could not connect to the server."]

The server is running successfully. I am sending the screenshots for reference.
https://ibb.co/nHZVtG
https://ibb.co/bMwM0w

Please help.
